tests/php/UsedOnTableTest/UsedOnTableExtension.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace SilverStripe\Admin\Tests\UsedOnTableTest;
+
+use SilverStripe\Assets\Shortcodes\FileLink;
+use SilverStripe\Core\Extension;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\ORM\DataObject;
+
+class UsedOnTableExtension extends Extension
+{
+    public function updateUsage(ArrayList &$usage, DataObject &$record)
+    {
+        $usage = $usage->exclude([
+            'ClassName' => [
+                FileLink::class,
```

Comment:
Should try and remove something other that FileLink since that's already removed by default https://github.com/silverstripe/silverstripe-asset-admin/blob/1.6/code/Extensions/UsedOnTableExtension.php

Even though it's on a different module, something may get confused when running this in the context of silverstripe/recipe-cms or kitchen-sink